packages/notification-center/src/i18n/languages/fr.ts

```diff
@@ -0,0 +1,10 @@
+import { ITranslationEntry } from '../lang';
+
+export const FR: ITranslationEntry = {
+  translations: {
+    notifications: 'Avis',
+    markAllAsRead: 'tout marquer comme lu',
+    poweredBy: 'Alimenté par',
```

Comment:
This works as a good translation but 'Propulsé par' sounds better.
```suggestion
    poweredBy: 'Propulsé par',
```